.travis.yml

```diff
@@ -1,15 +0,0 @@
-dist: trusty
-language: node_js
-node_js:
-  - "4"
-  - "6"
-  - "8"
-addons:
-  apt:
-    packages:
-      - google-chrome-stable
-env:
-  global:
-    - secure: ZaVxQhRv9HYXTCqeS9T4XXGO1cGayL11os8jDZ6wqNkQAwkOw3QSL70iloN5DMzg4WXv8Pwp5I4EUDIcUwiu4UCbDgfn1vYoTfVEmDMBDni4xw3CRG7IT8gEijJItCaOdEzC+IBmpAFuVI8xcuic7WBYPKRnShWkZ1xwmHDSenc=
-    - secure: lh7gaH4G4RUvdjrocupakeFbx4JtF7AyLGXGfsJl+/PJFjvrB+Ahatb8sTcb5hDHTr+Ucunzawx1HBNE+k+7CyykKBQpsQNWT/q6vlL/PAjRpt501O46NEdfTm/RSTqHgIo9WINYIa/ZpCh2L0vAevduaT0GGFCaP9W7fpJaVhM=
```

Comment:
not sure what these are but they don't seem to be required

Reply:
Looks like they were also unused in the original repo when we forked it